src/components/pool_flux_chart.vue

```diff
@@ -113,20 +121,33 @@ export default {
     },
     methods:{
         toggle() {
+          const self = this;
+
+          // Update global value for show Uncertainty
           this.showUncertainty = !this.showUncertainty;
+          
+          // Toggle on or off uncertainty bars
           if (this.showUncertainty) {
-            this.uncertaintyPrompt = 'Hide uncertainty'
+            this.uncertaintyPrompt = 'Hide ranges for estimates'
             this.d3.selectAll('.chartBandBkgd')
               .style("opacity", 1)
             this.d3.selectAll('.chartBand')
               .style("opacity", 0.3)
           } else {
-            this.uncertaintyPrompt = 'Show uncertainty'
+            this.uncertaintyPrompt = 'Show ranges for estimates'
```

Comment:
Nowhere are we saying what the default estimate is - mean, median?

Reply:
According to Abbott's paper:
> The central point [this is the e represents the most recent or comprehensive individual estimate, and error bars represent the range of reported values and their uncertainties

Reply:
interesting......

Reply:
definitely need to include that information

Reply:
K added.